### ISSUE TITLE
Need informative my_translations view

### ISSUE THREAD

**Issue by ViStefan** [duckduckgo/community-platform#33]
I'm translating ddg to ru_RU, and I want to see, how many phrases I've total translated, and how many from it are really helpful (top voted). It will be better, if we can see this information in "account". This idea can be realized like system of achievements for translators. E.g. "10 good translations achievement unlocked". Also it may be a reputation system (maybe like progress bar in "account"?).

**Comment by javathunderman** [comment]
@zekiel Can this be closed?

**Comment by zekiel** [comment]
edited the title for what's not completed yet. Right now, #79 covers the translations_in_use portion but the "helpfulness" indicator is more like "your translations ordered by # of votes"

**Comment by javathunderman** [comment]
Ok, great.

**Comment by zekiel** [comment]
@javathunderman  most basically, tho we're missing a my_translations view

**Comment by javathunderman** [comment]
Yeah, that was something I was meaning to mention. Any chance that may be coming soon?

**Comment by zekiel** [comment]
the whole platform is open source so you'll be able to see activity in the pull requests. The roadmap discussion is coming up w/ ComLeaders so we could discuss it there as well. Right now, though, we're not currently working on it. I imagine it could be wrapped in w/ the duck.co redesign planned for sometime early this year.

**Comment by javathunderman** [comment]
Yay redesign! :)
